src/workloads/networking/CommitLatency.yml

```diff
@@ -63,12 +62,14 @@ Actors:
 - Name: CommitLatency
   Type: CommitLatency
   Threads: 1
-  Repeat: 500
   Database: test
   Phases:
   - *Nop  # Load phase is different actor
   - *Nop
   - MetricsName: CommitLatency_w0_local
+    Collection: *Collection
```

Comment:
Yikes this turned out to be more duplication than I had imagined. I'm nervous about how users will feel.

What do you think about adding inheritance to HasNode's operator[]. I think it's a lot less risky and less code than adding inheritance to a more Node because you're not checking inherited keys at every level of the ["foo"]["bar"]["baz"] call chain.

I was thining of having the HasNode store a pointer to its parent and if operator[] returns an invalid node, it'll call operator[] on its parent.

Reply:
Ya I think this is a great idea.  But per convo, I'm going to not take you up on it.

For posterity:

1. The idea here is that we'd only really support inheriting from Phase to Actor to Workload and not from "Operation" to Phase since `Phase:Operation` in YAML isn't a thing Genny-proper knows about.
2. but that's *kinda* confusing since only some things inherit and CrudActor and RunCommand in particular use the `Operation` syntax
3. and we'd ultimately like to kill `ActorContext` and `PhaseContext` since they don't really provide much additional value and thus the `HasNode` shim here would eventually go away
4. *and* we'd like to eventually support a "Phase-first" syntax that lets you write workload YAMLs with the Phase being the top-level key and Actors inheriting from that.
5. Given that `genny::Node` is now *always* heap-allocated (or at least fully-owned by `NodeSource` and only exposed as const-ref) we could add back the full inheritance model where `Node`s have a pointer to their parents and do fallback there. We could add this back in if the configuration becomes too annoying without inheritance.

There's no concrete plans for the last 3 items but for now we're going to just *not* have inheritance. We'll add it back or consider the other 2 options if there's urgency from users.